mjk2244.md

```diff
@@ -0,0 +1,22 @@
+# Open Source Development
+
+Name: Matt Kim
+
+UNI: mjk2244
+
+Github: [link](https://github.com/mjk2244)  
+GitHub README: [link](https://github.com/timkpaine/timkpaine/blob/main/README.md](https://github.com/mjk2244/mjk2244/blob/main/README.md))
```

Comment:
`[link](https://github.com/timkpaine/timkpaine/blob/main/README.md](https://github.com/mjk2244/mjk2244/blob/main/README.md))`

malformed, and should be in separate PR for HW1